src/osa/scripts/tests/test_osa_scripts.py

```diff
@@ -397,3 +398,15 @@ def test_sequencer_webmaker(
     # Running without test option will make the script fail
     output = sp.run(["sequencer_webmaker", "-d", "2020-01-17"])
     assert output.returncode != 0
+
+
+def test_gainsel_webmaker(
+    base_test_dir,
+):
+
+    output = sp.run(["gainsel_webmaker", "-d", "20200117"])
+    assert output.returncode == 0
+    directory = base_test_dir / "OSA" / "GainSelWeb"
+    directory.mkdir(parents=True, exist_ok=True)
```

Comment:
I think this directory should be created by the script. No need to create it here.

Reply:
Changed